substrate/frame/parameters/src/lib.rs

```diff
@@ -167,6 +167,15 @@ pub mod pallet {
 		type WeightInfo: WeightInfo;
 	}
 
+	#[pallet::hooks]
+	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
+		fn on_initialize(_: BlockNumberFor<T>) -> Weight {
+			let items = Parameters::<T>::iter().count() as u64;
```

Comment:
Looking at the `support/procedural/src/dynamic_params.rs` in polkadot-sdk, I do not see any insert for the initial values. The storage will be empty and we should get he length from `T::RuntimeParameters::default()` instead.

Reply:
>  I do not see any insert for the initial values

We don't need to insert initial values in production, only in benchmark setup (for the on_initialize benchmark scenario that still need to be added in this PR)